e2etest/get_stats_test.go

```diff
@@ -0,0 +1,142 @@
+package e2etest
```

Comment:
Is there any automated way of running this?

Reply:
This can be tested via "go test". See the presubmit for this PR: https://screenshot.googleplex.com/CEeNOtwKQqg

Reply:
Oh nice, so this will run as part of presubmit?

Reply:
Yep.